Cargo.toml

```diff
@@ -8,6 +8,9 @@ edition = "2021"
 [dev-dependencies]
 uuid = {version = "1.0.0", features=["v4"]}
 
+[dependencies]
+clap = { version = "3.1.18", features = ["derive"] }
```

Comment:
Clap is nice but extremely bloated. Look at the cargo.lock stuff it pulled in. There are a lot of other nice minimal command line argument libraries in Rust that should be by far enough until we rewrite the compiler in jakt.

Reply:
Thanks for adding a suitable pico-args implementation! I need more rust under my belt to figure out a "clean" solution for using that.